### PR TITLE
TW-159: Add how-to for creating support tickets

### PR DESCRIPTION
**Pull request by djmagee**
## What does this PR do?

- Adds a page about how to create a ticket for Temporal Support.
- Links to page from seven places across three other pages.

## Notes to reviewers

I opted not to provide details for most optional fields in the default form because the form provides explanations and ghost text.

### CODE REVIEW

File: docs/cloud/how-to-create-a-ticket-for-temporal-support.md [hunk 1]
```diff
@@ -0,0 +1,24 @@
...
+description: To request assistance from Temporal Support, create a ticket in Zendesk.
+tags:
+  - explanation
+---
+
+To request assistance from Temporal Support, create a ticket in Zendesk:
+
+1. Go to [support.temporal.io](https://support.temporal.io/).
+1. Use your Temporal Support credentials to sign in.
+   If you don't have a Temporal Support account, choose **Sign up**.
```
Comment (flossypurse) [inline docs/cloud/how-to-create-a-ticket-for-temporal-support.md:14]: **you mush be a Temporal Cloud customer to do this**

File: docs/cloud/how-to-create-a-ticket-for-temporal-support.md [hunk 2]
```diff
@@ -0,0 +1,24 @@
...
+1. Choose **Create a ticket**.
+1. On the **Submit a request** page, choose your issue.
+   Unless your request involves one of the specific areas listed, choose **Default Ticket Form**.
+1. In the form, enter the details of your request.
+   **Subject** and **Description** are required.
+1. If you specify **Priority** (available only on the default form), follow these guidelines:
+   - Select **Normal** for most issues.
+   - Select **High** only for issues to which your Service Level Agreement (SLA) applies.
```
Comment (flossypurse) [inline docs/cloud/how-to-create-a-ticket-for-temporal-support.md:22]: **Check with your org to find out what that is**

**Review by flossypurse — APPROVED** [review]

File: docs/cloud/how-to-create-a-ticket-for-temporal-support.md [hunk 3]
```diff
@@ -0,0 +1,31 @@
...
+1. Use your Temporal Support credentials to sign in.
+1. Choose **Create a ticket**.
+1. On the **Submit a request** page, choose your issue.
+   Unless your request involves one of the specific areas listed, choose **Submit a Ticket**.
+1. In the form, enter the details of your request.
+   **Subject** and **Description** are required.
+1. If you specify **Priority** (available only on the default form), follow these guidelines:
+   - Select **Normal** for most issues.
+   - Select **High** only for issues to which your service-level agreement (SLA) applies.
+     If you're not sure, check with your Temporal administrator.
```
Comment (tsurdilo) [inline docs/cloud/how-to-create-a-ticket-for-temporal-support.md:29]: whats a temporal administrator? an internal person to the customer or someone at temporal?

Comment (djmagee) [reply docs/cloud/how-to-create-a-ticket-for-temporal-support.md:29]: Presumably someone at the customer company who has knowledge of the applicable SLA. What would be a clearer way to express this? Or should I remove the sentence?

File: kb/deadline-exceeded-troubleshooting.md [hunk 4]
```diff
@@ -22,7 +22,7 @@ The following sections discuss the nature of this error and how to troubleshoot
 
 Cloud users cannot access some of the logs needed to diagnose the source of the error.
 
-If you're using Temporal Cloud, check your Workflow History for error messages and create a [support ticket](https://support.temporal.io/) with Temporal.
+If you're using Temporal Cloud, check your Workflow History for error messages and create a [support ticket](https://docs.temporal.io/cloud/how-to-create-a-ticket-for-temporal-support).
```
Comment (tsurdilo) [inline kb/deadline-exceeded-troubleshooting.md:25]: I'd remove the "check your workflow history for error message". maybe better to say here "create a ticket and provide as much information as possible, including namespace name and workflow id(s) of some executions experiencing the issue" or something similar

**Review by jreynolds232 — APPROVED** [review]

File: kb/deadline-exceeded-troubleshooting.md [hunk 5]
```diff
@@ -110,4 +110,4 @@ If you increase `frontend.keepAliveMaxConnectionAge` values, consider monitoring
 
 ---
 
-If you were unable to resolve your issue, check for similar questions and possible solutions on our [community forum](https://community.temporal.io) or [community Slack](https://temporal.io/slack), or file a [support ticket](https://support.temporal.io/).
+If you were unable to resolve your issue, check for similar questions and possible solutions on our [community forum](https://community.temporal.io) or [community Slack](https://temporal.io/slack), or file a [support ticket](https://docs.temporal.io/cloud/how-to-create-a-ticket-for-temporal-support).
```
Comment (tsurdilo) [inline kb/deadline-exceeded-troubleshooting.md:113]: If this is specific to cloud, i'd put open support ticket first before recommending community resources to customers, but of course they can check those too

File: cloud/release-notes/release-note-2022-09-21.md [hunk 6]
```diff
@@ -51,4 +51,4 @@ Namespace deletion isn't supported at this time. The feature is in active develo
 ### Namespace quota
 
 By default, each account has a quota of 10 Namespaces.
```
Comment (tsurdilo) [inline cloud/release-notes/release-note-2022-09-21.md:53]: quota=limit?

File: docs/cloud/how-to-create-a-ticket-for-temporal-support.md [hunk 7]
```diff
@@ -0,0 +1,31 @@
...
+sidebar_label: Support ticket
+description: To request assistance from Temporal Support, create a ticket in Zendesk.
+tags:
+  - explanation
+---
+
+:::info
+
+This procedure applies only to Temporal Cloud customers whose contracts include paid support.
+If you need assistance and don't have paid support, post your request in the [Temporal Community Forum](https://community.temporal.io) or the [Temporal Slack workspace](https://t.mp/slack).
```
Comment (tsurdilo) [inline docs/cloud/how-to-create-a-ticket-for-temporal-support.md:13]: we have a specific #support-cloud channel that we might want to mention here
idk if we should put "paid support" here or not as iirc all customers have "paid support" by default. would check with sales team on this please

File: docs-src/cloud-context/namespaces-create.md [hunk 8]
```diff
@@ -46,7 +46,7 @@ To create a Namespace in Temporal Cloud, gather the following information:
 1. In **Retention Period**, specify a value from 1 to 90 days.
    When choosing this value, consider your needs for Event History versus the cost of maintaining that Event History.
    Typically, a development Namespace has a short retention period and a production Namespace has a longer retention period.
```
Comment (tsurdilo) [inline docs-src/cloud-context/namespaces-create.md:48]: I don't think we distinguish between this, and all namespaces have a default retention period when they are created. @antmendoza can you confirm please?

File: docs/cloud/how-to-create-a-ticket-for-temporal-support.md (hunk 3 above)
Comment (djmagee) [reply docs/cloud/how-to-create-a-ticket-for-temporal-support.md:29]: Let's bypass the issue and replace the sentence with "If you're not sure, select **Normal**." Either way, Support gets the ticket, and you can elevate it if needed.

File: docs/cloud/how-to-create-a-ticket-for-temporal-support.md (hunk 7 above)
Comment (djmagee) [reply docs/cloud/how-to-create-a-ticket-for-temporal-support.md]: Added name of channel. Checking with Sales.

File: docs-src/cloud-context/namespaces-create.md (hunk 8 above)
Comment (djmagee) [reply docs-src/cloud-context/namespaces-create.md:48]: When you use the UI to create a Namespace, the slider for Retention Period defaults to 30 days.

If we need to update this section, let's create a separate task.

Comment (antmendoza) [reply docs-src/cloud-context/namespaces-create.md:48]: @tsurdilo as @djmagee states, the default value in the form is 30 days.

**Pull request merged**